### ISSUE TITLE
executed commands run twice?

### ISSUE THREAD

**Issue by joedefen** [hluk/CopyQ#2306]
**Describe the bug**
When a string starting with "ExEc=" is copied to the keyboard, I wish that whole text be run once as a bash command, but it being run twice.  Probably, it is a pilot error, but I don't see any controls to fix this.

**To Reproduce**
1. In "+ Add" command..
2. Choose "Automatic"
3. Enter Name: Run ExEc2
4. Advanced: Content: ^ExEc=
5. Click: Apply
6. Command: copyq:  execute('bash', null, input())

**Expected behavior**
When I copy something to the keyboard, e.g., "ExEc= mpv movie.mp4", I expect mpv to be run once, but it is launched twice.

**Screenshots**

**Version, OS and Environment**
CopyQ Clipboard Manager 6.0.1
Qt: 5.15.2
KNotifications: 5.89.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: KDE neon 5.27

**Additional context**
I'm stumped.

**Comment by hluk** [comment]
Yup, automatic commands run on both selection and clipboard by default. This can be distinguished in the commands using [isClipboard()](https://copyq.readthedocs.io/en/latest/scripting-api.html#isClipboard) function.

```js
copyq:
if (!isClipboard()) {
    abort();
}
...
```
